OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemoryManager.cs

```diff
@@ -36,7 +35,7 @@ private void FindMemory()
             }
             else if (repository.GetMachinaRegion() == GameRegion.Korean)
             {
-                candidates.Add(container.Resolve<ICombatantMemory60>());

```

Comment:
If `InCombatMemory60` is now unused, please delete the file.

Reply:
The `CombatMemory60.cs` file is still present?

Reply:
oh, sorry. fixed.